pkg/descriptions.json

```diff
@@ -1405,6 +1405,11 @@
         ]
     },
     {
+        "Name": "System.Security.Permissions",
+        "Description": "Provides types supporting Code Access Security (CAS).",
```

Comment:
Do we have a standard way of discouraging use of a package? People really shouldn't use this for anything other than making old code compile/run given that it doesn't implement anything.